conda.recipe/build.sh

```diff
@@ -0,0 +1 @@
+python setup.py install
```

Comment:
I think we can probably get everything defined in just the `meta.yaml`, no need for separate build script.

Reply:
Done